.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,23 @@
+name: Push Image
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout secret-manger
+        uses: actions/checkout@v2
+        with:
+          path: secret-manager
+      - name: Build and push Docker images to quay
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+          repository: xxxxxxxx/secret-manager
```

Comment:
TODO: need repositories and secrets

Reply:
Will make repositories in the AM and get everyone added. I dont know that there is a _right_ way for github secrets as there is no jwt auth method and something has to be stored. I just did it the vault way out of familiarity. Did you have another idea?

Reply:
Oh, i think my point got lost in translation :/ 
I don't care where credentials are stored, vault is absolutely fine. Since we're using GitHub Actions we might consider using GitHub secrets aswell: they integrate quite nicely and there are less dependencies.

Reply:
@moolen 
- [x] quay.io repository + robot username/token created (Uses above example for keys)
- [x] dockerhub repository + robot account/token created (Uses `DOCKERHUB_*` syntax)
- [x] github container registry + robot account/token created (Uses `GHCR_*` syntax)